api/pkg/apps/cms/case_post.go

```diff
@@ -15,7 +15,9 @@ func (s *Server) PostCase(w http.ResponseWriter, req *http.Request) {
 	}
 
 	kase := &payload
-	kase.ID = uuid.NewV4().String()
+	if len(kase.ID) == 0 {
+		kase.ID = uuid.NewV4().String()
+	}
```

Comment:
```suggestion
	kase.ID = uuid.NewV4().String()
```
All IDs have to be generated by the server ;)